pkg/autoupdate/tuf/ci/tuf_server.go

```diff
@@ -163,11 +163,15 @@ func compress(t *testing.T, outFileName string, outFileDir string, targetDir str
 	defer tw.Close()
 
 	srcFilePath := binary
-	if binary == "launcher" && runtime.GOOS == "darwin" {
-		srcFilePath = filepath.Join("Kolide.app", "Contents", "MacOS", binary)
+	if runtime.GOOS == "darwin" {
+		appBundleName := "Kolide.app"
+		if binary == "osqueryd" {
+			appBundleName = "osquery.app"
+		}
+		srcFilePath = filepath.Join(appBundleName, "Contents", "MacOS", binary)
```

Comment:
```suggestion
		srcFilePath = filepath.Join(appBundleName, "Contents", "MacOS", binary)
		
```

Reply:
Sorry, I applied this and then realized it resulted in two empty newlines, so I've undone the change. Was there something else you wanted to update here?

Reply:
There already was one, that's why lint failed

Reply:
Do you want a new line before 171 and not after it?

Reply:
Before 171, but it doesn't matter much. My thinking is that you have:
```
		appBundleName := "Kolide.app"

		if binary == "osqueryd" {
			appBundleName = "osquery.app"
		}
		srcFilePath = filepath.Join(appBundleName, "Contents", "MacOS", binary)
```

which grouped things into:
1. set appBundleName
2. change appBundleName, and set srcFilePath

and that just seems weird to me. So I'd shuffle the newlines to feel more grouped. But 🤷